app/models/case_log.rb

```diff
@@ -254,15 +256,12 @@ def get_totadult
     end
   end
 
-  def get_la(postcode)
+  def get_inferred_la(postcode)
     postcode_lookup = nil
     Timeout.timeout(5) { postcode_lookup = PIO.lookup(postcode) }
     if postcode_lookup && postcode_lookup.info.present?
-      self.is_la_inferred = true
```

Comment:
I think this might be needed (probably line 265 too), cause if we consider this situation:
1. Enter a valid postcode
2. LA gets inferred
3. Go back and change `postcode_known` to `No`

I don't think that LA would get reset

Reply:
Hmm, never mind that. It actually gets reset 👀